src/test/java/core/basesyntax/strategy/impl/PurchaseOperationCalculatorImplTest.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy.impl;
+
+import static junit.framework.TestCase.assertEquals;
+
+import core.basesyntax.strategy.OperationCalculator;
+import org.junit.Test;
+
+public class PurchaseOperationCalculatorImplTest {
+    private static final int VALID_BALANCE = 20;
+    private static final int VALID_QUANTITY = 13;
+    private static final int EXPECTED_RESULT = VALID_BALANCE - VALID_QUANTITY;
+    private OperationCalculator purchaseOperationCalculator = new PurchaseOperationCalculatorImpl();
+
+    @Test
+    public void calculatePurchase_isOk() {
```

Comment:
add negative tests

Reply:
The method has two input int values and does one operation with them. What negative tests do you mean? There were no checks for a positive balance in the fruit shop technical requirements, if that's what you mean. Or are you talking about something else? This comment is for all Operation classes

Reply:
I mean tests with null value balance, null value quantity, null value both values, 0 values